src/api/application/application.ts

```diff
@@ -140,13 +140,20 @@ export default class ApplicationModule extends Base {
         await this.wire.sendAction('create-application', appOptions);
         return await this.wrap({ uuid: appOptions.uuid });
     }
+
+    /**
+    * DEPRECATED method to create a new Application. Use {@link Application.start} instead.
+    * @param { ApplicationOption } appOptions
+    * @return {Promise.<Application>}
+    * @tutorial Application.create
+    */
     public create(appOptions: ApplicationOption): Promise<Application> {
         console.warn('Deprecation Warning: fin.Application.create is deprecated. Please use fin.Application.start');
         return this._create(appOptions);
     }
     /**
     * Creates and starts a new Application.
-    * @param { ApplicationOption } appOptions
+    * @param { Application~options } appOptions
```

Comment:
ApplicationOption is used in DefinitelyTyped. Maybe we should change Application~options to ApplicationOption in jsdoc.

Reply:
Thank you @licui3936. 
Q: Why are the [type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/openfin/_v2/api/application/applicationOption.d.ts) names for App & Win options are in singular form? doesn't it make more sense to use the plural form? (i.e ApplicationOptions)

Reply:
Sounds good to me. If we change it, we also need to update it in DefinitelyTyped.